docs/src/Theory/Atmos/EDMFEquations.md

```diff
@@ -143,29 +143,41 @@
 \newcommand{\LatentHeatF}[1]{L_{fus}(#1)}
 ```
 
-# Eddy-Diffusivity Mass-Flux (EDMF) equations
-
-This document is concerned with defining the set of equations solved in the atmospheric turbulence convection model: the EDMF equations. Color-coding is used to indicate:
-
+# Eddy-Diffusivity Mass-Flux (EDMF) parameterization
+
+This document describes the Eddy-Diffusivity Mass-Flux (EDMF)
+parameterization for sub grid scale (SGS) turbulance and convection.
+the model equations and rational are based on:
+Tan et al. (2018); Cohen et al. (2020); Lopez-Gomez et al. (2020)
+and Jia et al. (2020).
+The key predictents of the EDMF parameterization are the SGS vertical fluxes of heat moisture
+and momentum, and the cloud fraction in the host model grid.
+The EDMF parameterization divides the host model's grid into $N \ge 2$ subdomain
+that represents a isotropic turbulent enviroment and coherent updraft(s)
+and or downdafts. The parameterization solves prognostic equations
```

Comment:
**and or downdaft(s).**

Reply:
I'm assuming this is supposed to be "downdraft(s)"